socialdistribution/app/views.py

```diff
@@ -426,12 +426,27 @@ def sync_github_activity(request):
 
 def github_event_to_post_adapter(author, event):
     type = Post.ContentType.PLAIN
-    title = f"Github Event of type {event['type']}"
+    title = f"Github: {event['type']}"
     categories = f"github, {event['type']}"
+    try:
+        head = str(event['payload']['head'])
+    except:
```

Comment:
Could you only except a specific exception? I think here it would be key not found